calabash-cucumber/scripts/.irbrc

```diff
@@ -47,3 +43,22 @@ extend Calabash::Cucumber::Operations
 def embed(x,y=nil,z=nil)
   puts "Screenshot at #{x}"
 end
+
+require "calabash-cucumber"
+
+def preferences
+  @preferences ||= Calabash::Cucumber::Preferences.new
+end
+
+def disable_usage_tracking
+  preferences.usage_tracking = "none"
+  puts "Calabash will not collect usage information."
+  "none"
+end
+
+def enable_usage_tracking(level="system_info")
+  preferences.usage_tracking = level
```

Comment:
That's sort of what I mean - is it not possible someone could `enable_usage_tracking` while usage_tracking is being read from disk? Perhaps it's not, but as-is I don't think it would 'fail or block', at least not gracefully (unless ruby `io` does that by default).